examples/tv-app/linux/include/account-login/AccountLoginManager.cpp

```diff
@@ -43,9 +48,19 @@ bool AccountLoginManager::HandleLogout()
 Commands::GetSetupPINResponse::Type AccountLoginManager::HandleGetSetupPin(const chip::CharSpan & tempAccountIdentifier)
 {
     string tempAccountIdentifierString(tempAccountIdentifier.data(), tempAccountIdentifier.size());
-    ChipLogProgress(Zcl, "temporary account id: %s", tempAccountIdentifierString.c_str());
-    // TODO: Insert your code here to handle get setup pin
+
     Commands::GetSetupPINResponse::Type response;
-    response.setupPIN = chip::CharSpan("tempPin123", strlen("tempPin123"));
+    if (mSetupPIN == 0)
```

Comment:
Why not set a dummy pin at AccountLoginManager? How could we get a real pin？

Reply:
a custom app implementation (an android app) would call a proprietary cloud method to get the pin